config.md

```diff
@@ -265,6 +265,27 @@ Annotations are key-value maps.
 }
 ```
 
+## HypervisorPath
+
+HypervisorPath is an optional absolute path to the hypervisor binary used by VM-based runtimes.
+
+## ImagePath
+
+ImagePath is an optional absolute path to the root filesystem image used by VM-based runtimes.
```

Comment:
`ImagePath` is the path of the image outside the vm. ImageMountPoint is the path inside the vm (the path is chosen by the runtime-implmentation) for mounting the image.
The image can't not be mounted in the host (if the fs inside the image is not disbuted filesystem) when the container is running, it is mount only in the vm. so there is not mount-point in the host and there is no way to "find a block-device mount-point".
And the root directory inside the image may not be the rootfs of the container, so root.path is needed for pointing out where is the rootfs for the container. See the devicemapper driver of docker.